lib/utils/app-installation.ts

```diff
@@ -32,4 +34,124 @@ async function isAppInstalled(
   }
 }
 
-module.exports.isAppInstalled = isAppInstalled
+export async function installApp(
+  client: ClientAPI,
+  {
+    spaceId,
+    environmentId,
+    appId
+  }: {
+    spaceId: string
+    environmentId: string | string[]
+    appId: string
+  }
+): Promise<void> {
+  const environments = Array.isArray(environmentId)
+    ? environmentId
+    : [environmentId]
+
+  for (const environmentId of environments) {
+    await client.rawRequest({
+      method: 'PUT',
+      url: `/spaces/${spaceId}/environments/${environmentId}/app_installations/${appId}`,
+      data: {
+        parameters: {}
+      }
+    })
+  }
+}
+
+const promptAppInstallationInEnvironment = async (
+  client: ClientAPI,
+  spaceId: string,
+  environmentId: string,
+  appId: string
+) => {
+  warning(
+    `The Merge app is not installed in the environment with id: ${environmentId}`
+  )
+
+  const userConfirmation = await confirmation(
+    `Do you want to install the Merge app in the environment with id: ${environmentId}`
+  )
+
+  if (userConfirmation) {
+    return false
+  }
```

Comment:
Should this be checking for `!userConfirmation`?

Reply:
Yes! When testing things for the Show & Tell I came across this.